eZ/Publish/Core/settings/limitations/language.yml

```
@@ -0,0 +1,12 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+    _instanceof:
+        eZ\Publish\Core\Limitation\LanguageLimitation\VersionTargetEvaluator:
+            tags:
+                - { name: ezplatform.core.permissions.limitations.language.version_target_evaluator }
```

IMHO `ezplatform.core.permissions.limitations.language.version_target_evaluator` is too long. How about    

```suggestion
                - { name: ezplatform.limitation.language_version_target_evaluator }
```

?